events/Gondor.txt

```diff
@@ -2568,7 +2566,7 @@ country_event = {
 		prestige = -10
 		add_country_modifier = {
 			name = harondor_lost
-			duration = 4380
+			duration = -1
```

Comment:
If you want to go this route, can the modifier be removed if Harondor is reclaimed?

Reply:
It can be removed if Harondor is reclaimed through Acquire State or some other means. I also think that a Good unified Harad could make some treaty with Gondor to keep Harondor that will remove this modifier and potentially put another modifier in it's place.

Reply:
Fair enough. Just wanted to make sure that there is some way to remove it should Gondor's fortunes change -- it wouldn't make sense for them to have the modifier if they reconquered it